public/css/module.less

```diff
@@ -44,9 +44,17 @@ div.cube {
 
   div.cube-dimension2 {
     > .header {
+      display: flex;
       text-align: center;
       color: white;
       margin-bottom: 1em;
+
+      > a:first-child {
+        flex: 1;
```

Comment:
Using `display: block;` here is sufficient and does not require flex.

Reply:
The flex here is to properly align the filter icon to the right while maintaining a set width to truncate on :)

Reply:
Check. Makes sense. But it must also work for `cube-dimension1`.

Reply:
`.cube-dimension-1` doesn't need this, because it's nothing more than a headline with no content and there is nothing that gets pushed down - so i think it would make more sense to allow longer headings there.

Reply:
Please configure 3 dimensions 😉